Client/Frontend/Browser/BrowserViewController/OpenSearch/OpenSearchEngineButton.swift

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import SnapKit
+import BraveShared
+
+// MARK: - OpenSearchEngineButton
+
+class OpenSearchEngineButton: UIView {
```

Comment:
Custom button should probably just be a `UIControl` subclass instead of a `UIView` with a button inside of it or consider using `Button` in `BraveUI` framework which includes the ability to set a loader

Reply:
UIView is converted to Button and loaderView is instantiated properly. Fix at https://github.com/brave/brave-ios/pull/3231/commits/ba493be2def981cae70ab0ce8231326645c31a0d